widget/view.controller.js

```diff
@@ -366,49 +409,128 @@
                 });
                 return;
             }
             else if (!CommonUtils.isUndefined(threatHuntConfigForm.qradarForm) && threatHuntConfigForm.qradarForm.$invalid) {
                 var huntToolIndex = $scope.selectedEnv.huntTools.indexOf('IBM QRadar');
-                _activeErrorTab(huntToolIndex, 'IBM QRadar');
+                _activeErrorTab('IBM QRadar', huntToolIndex);
                 toaster.error({
                     body: 'IBM QRadar Threat Hunt Tool parameters are required'
                 });
                 return;
             } else if (!CommonUtils.isUndefined(threatHuntConfigForm.splunkForm) && threatHuntConfigForm.splunkForm.$invalid) {
                 var huntToolIndex = $scope.selectedEnv.huntTools.indexOf('Splunk');
-                _activeErrorTab(huntToolIndex, 'Splunk');
+                _activeErrorTab('Splunk', huntToolIndex);
                 toaster.error({
                     body: 'Splunk Threat Hunt Tool parameters are required'
                 });
                 return;
             } else {
-                $scope.selectedEnv.installOutbreakType = $scope.outbreakAlertSeverityList.slice();
-                WizardHandler.wizard('OutbreaksolutionpackWizard').next();
+                $scope.isConnectorsHealthy = true;
+                // Array to hold all promises
+                let promises = [];
+
+                for (let index = 0; index < $scope.selectedEnv.huntTools.length; index++) {
+                    let huntToolName = _.get($scope.huntToolsMapping, $scope.selectedEnv.huntTools[index]);
+                    let queryBody = {
+                        "logic": "AND",
+                        "filters": [
+                            {
+                                "field": "name",
+                                "operator": "in",
+                                "value": huntToolName
+                            }
+                        ]
+                    };
+
+                    // Create a promise for each API call
+                    let promise = $resource(API.QUERY + 'solutionpacks').save({ $limit: ALL_RECORDS_SIZE }, queryBody).$promise
+                        .then(function (response) {
+                            if (Array.isArray(response['hydra:member']) && response['hydra:member'].length > 0) {
+                                let huntToolDetails = _.map(response['hydra:member'], obj => _.pick(obj, ['name', 'label', 'version', 'uuid']));
+                                return connectorService.getConnector(huntToolDetails[0].name, huntToolDetails[0].version)
+                                    .then(function (connector) {
+                                        if (!connector) {
+                                            toaster.error({
+                                                body: 'The Connector "' + huntToolDetails[0].name + '" is not installed. Install the connector and re-run this wizard to complete the configuration'
```

Comment:
Install the connector and re-run --> did you mean to say "Install the connector manually and re-run"?

Reply:
yes .. made changes